src/standby.h

```diff
@@ -30,12 +34,31 @@ void updateStandbyTimer(void) {
         }
         else {
             standbyModeRemainingTimeMillis = 0;
+            LOG(INFO, "Entering standby mode...");
+        }
+    }
+    else if ((standbyModeRemainingTimeDisplayOffMillis != 0) && ((currentTime % 1000) == 0) && (currentTime != lastStandbyTimeMillis)) {
+        unsigned long standbyModeTimeMillis = (standbyModeTime * 60 * 1000) + (TIME_TO_DISPLAY_OFF * 60 * 1000);
+        long elapsedTime = currentTime - standbyModeStartTimeMillis;
+        lastStandbyTimeMillis = currentTime;
+
+        if (standbyModeTimeMillis > elapsedTime) {
+            standbyModeRemainingTimeDisplayOffMillis = standbyModeTimeMillis - elapsedTime;
+
+            if ((currentTime % 60000) == 0) {
+                LOGF(INFO, "Standby time until display is turned off: %i minutes", standbyModeRemainingTimeDisplayOffMillis / 60000);
+            }
+        }
+        else {
+            standbyModeRemainingTimeDisplayOffMillis = 0;
+            LOG(INFO, "Turning off display...");
         }
     }
 }
 
 void resetStandbyTimer(void) {
-    standbyModeRemainingTimeMillis = standbyModeTime * 60 * 10000;
+    standbyModeRemainingTimeMillis = standbyModeTime * 60 * 1000;
```

Comment:
How does change existing settings, will everything be x10 faster?

Reply:
Can't even remember making that change 😄
Definitely doesn't make any sense...

Reply:
I guess no one noticed that the reset always made the next standby 10x as long ;)